docs/rst/recording/usage/configuration.rst

```diff
@@ -560,6 +560,17 @@ QoS
 
     The :ref:`Topic QoS <recorder_topic_qos>` configured in ``specs`` can be overwritten by the :ref:`Manual Topics <recorder_manual_topics>`.
 
+
+Logging Configuration
+^^^^^^^^^^^^^^^^^^^^^
+
+The ``specs`` section supports an optional ``logging`` configuration to set up logging via YAML.
```

Comment:
Please rewrite the documentation. Try to use words that make sense to a user who hasn't read the code, add a table with the default values (there are tables throughout the documentation you can get inspiration from), and don't be afraid to use notes and warnings if necessary.

Reply:
``specs`` supports a ``logging`` **optional** tag to configure the |ddsrecorder| logs.

Reply:
I would also rather refer to the new configuration as `Configuring the logs via the Command-Line` rather than `Logging configuration via Command-Line`.

Reply:
Please:
1. shorten the description of the verbosity inside the table,
2. set correctly the data type of the verbosity.

Reply:
Remove `Set the verbosity to` from the description of the verbosity inside the table. It's already implied and it will make the description even shorter.